src/Layers/FeatureLayer/FeatureLayer.js

```diff
@@ -98,7 +98,16 @@ EsriLeaflet.Layers.FeatureLayer = EsriLeaflet.Layers.FeatureManager.extend({
       if(!layer){
         newLayer =  this.createNewLayer(geojson);
         newLayer.feature = geojson;
-        newLayer._originalStyle = this.options.style;
+
+        if (this.options.style) {
+          newLayer._originalStyle = this.options.style;
+        }
+
+        // circleMarker check
+        if (newLayer.setStyle) {
```

Comment:
This would really just trap anything with `setStyle` including `L.Polygon`, `L.Path`, ect... which is probably something we want to do anyway.